source/grid/manifold_lib.cc

```diff
@@ -1127,6 +1161,32 @@ FunctionManifold<dim,spacedim,chartdim>::~FunctionManifold()
 
 
 
+template<int dim, int spacedim, int chartdim>
+std::unique_ptr<Manifold<dim, spacedim> >
+FunctionManifold<dim,spacedim,chartdim>::clone() const
+{
+  if (owns_pointers == true)
```

Comment:
What does `owns_pointer` refer to? I forgot -- could probably look it up, but it's sufficiently obscure that a one-line comment might be useful :-)